### ISSUE TITLE
Any particular reason why ABS material?

### ISSUE THREAD

**Issue by arcusfelis** [GliaX/tourniquet#14]
Material question.

PLA or PETG are more common and easier to print. Any particular reason why it should be avoided?

**Comment by xchg-dot-ca** [comment]
PLA is a bit more brittle
PETG probably better choice here also easier to print. Crappy ABS prints with poor layer adhesion will be way worse then PETG or even PLA.
But again there are crappy PETG brands as well. So for such application I would use well tested plastic brand that prints well on your printer.
Carbon fiber filled nylon is probably the best choice ( providing you can print it and find in colors other than black )

**Comment by pierrotsmnrd** [comment]
Hi everyone. Quick questions related to the material : 

1. What are the required properties for the material to print tourniquets ? resistant but not brittle ? slightly flexible ? 
2. Would ASA also be a suitable material ? 
3. I have a left-over spool of Prusa PETG, so I intend on experimenting with this first before buying any other filament. My question is : Has there been 3D printed tourniquets in PETG already produced and successfully used without problem ? 

If so, it's worth I experiment and then produce in PETG (more convenient for me, easier to print), otherwise I'll switch to any other recommended material (ABS, or ASA if suitable as well)

I'm based in Strasbourg, France, and a local Ukrainian association named [PromoUkraina](https://promoukraina.fr/) makes convoy to Ukraine to provide with collected food and medicines. I'm already in touch with them, I know they could also send the tourniquets. I'm very motivated in producing them.

**Comment by BobBaumeisterXL** [comment]
Great Projekt!
Asa and Petg shoud also do Job.
I am not realy sure if an infill of 100% is needed, we will do some tests.
The heatbed temp for abs shoud be an little bit hihger  something around 110 degree.
I also created an version with brimears wich makes it possible to print it much faster.
Feel free to contact me if u need an copy.

**Comment by talentedmrripley** [comment]
This article from a member of the Glia team has some info on the material selection - https://trklou.medium.com/reinventing-3d-printed-tourniquets-for-ukraine-is-a-mistake-be2937f0ad65

**Comment by arcusfelis** [comment]
Good article.

ok, so for back plate should be flexible and don't break (PETG makes sense).

PLA is not ok, because of storage under sun and high temps - which is kinda important for Gaza. Also, in humid conditions. It is good idea to seal produced tourniquets into some kind easy to unpack plastic bag anyway.

Windlass _probably_ could be from PLA, because it would be hard to break anyway - it does not have small elements. But there is no reason just to use PETG anyway.

The other question is about 100% infill. 75% geroid is producing stronger prints (especially visible on the clip). But geroid makes less flexible backplate (which is equal to making it thicker). 

Also, cooling when printing. Use as little cooling when printing as possible (important both for PLA and PETG). Makes a stronger build.

**Comment by tareko** [comment]
Hello friends! Although I've been active with commits the past little while, I have neglected the issues. Notifications are now fully on, so I shouldn't miss more.

TL;DR: **NO PLA**, **YES PETG**, **YES ABS**

Thank you all for the great discussion. As noted above by @arcusfelis , our environment in Gaza required a hearty plastic, and recycled ABS is readily available there. As @xchg-dot-ca noted, badly-printed or low quality ABS is a huge problem. It took us time to get the settings and the quality control (QC) right on these.

We completed structural tests and PETG is a plastic we can now recommend. We'll publish the details in the coming weeks once we properly analyze and write it up. We haven't tested Nylon, ASA, etc., and don't recommend them until we can properly test them.
